Public/Invoke-DockerBuild.ps1

```diff
@@ -22,5 +22,11 @@ function Invoke-DockerBuild {
         $File = "Dockerfile"
     )
     $postfixedRegistry = Add-Postfix -Value $Registry
-    Invoke-Command "docker build `"${Context}`" -t ${postfixedRegistry}${ImageName}:${Tag} -f `"${File}`""
+    $commandResult = Invoke-Command "docker build `"${Context}`" -t ${postfixedRegistry}${ImageName}:${Tag} -f `"${File}`""
+    $result = [PSCustomObject]@{
+        "Dockerfile" = $File;
```

Comment:
Curious, any use case to return `Dockerfile`? Except maybe for linting?

Reply:
To support use of it further down the pipeline.